lib/WP_Auth0_Configure_JWTAUTH.php

```diff
@@ -3,14 +3,14 @@
 /**
  * Class WP_Auth0_Configure_JWTAUTH
  *
- * @deprecated 3.10.0, plugin is deprecated and removed from the WP plugin repo.
+ * @deprecated - 3.10.0, plugin is deprecated and removed from the WP plugin repo.
```

Comment:
Is the `-` a formatting requirement ? I'll approve anyway, just curious.

Reply:
No, just inconsistent with the rest